src/chocolatey.resources/helpers/functions/Install-ChocolateyShortcut.ps1

```diff
@@ -89,7 +89,7 @@ Install-ChocolateyShortcut -ShortcutFilePath "C:\test.lnk" -TargetPath "C:\test.
 Install-ChocolateyShortcut `
   -ShortcutFilePath "C:\notepad.lnk" `
   -TargetPath "C:\Windows\System32\notepad.exe" `
-  -WorkDirectory "C:\" `
+  -WorkingDirectoryq "C:\" `

```

Comment:
Note there is still a typo here "q" got into the end of this. Got to hit that escape key first, then q to get out of vim. ;)